book/proposal/outline.md

```diff
@@ -0,0 +1,125 @@
+
+This leads to a series of problems with the solution, a lack of domain knowledge throughout the organization, misleading code, poor data structures, un-maintainable codebase, insufficient or incorrect solutions and worse.
+
+We will explore the intersection of the fields of design thinking, agile and domain-driven design as a better way to build products.
+
+In this first chapter, we will take a look at what design thinking is, outline the commonly referred to process and pick apart whether it needs to be done in totality. 
+
+Design thinking process can offer an excellent way to engage domain experts in the creation of a product. Domain-driven design offers a set of tools for taking these findings and creating software implementations that more concisely capture the knowledge we have learned.
+
+In this chapter we will introduce domain driven design ideas we will be covering in this book: 
+
+- Ubiquitous languages
+- Glossaries
+- Context boundaries for primary and supporting domains
+- Domain events
+- Org personas
+
+## Chapter 2: Discovery
+We will outline the steps a development team is taking to implement a new warranty tool for a bike company. Working on any problem first requires learning everything you can about the problem. This critical phase is often one that software developers are left out of. We will explore why this is and why it should not be the case. 
+
+The first exercise introduced is a Stake Holder Mapping. This is a method for identifying potential domain experts to consult regarding the current process in place for warranty issues. The StakeHolder mapping also serves as a nice first pass at a bounded context diagram.
+
+Next, we will introduce how to develop buy-in from domain experts and bring them into the process. This includes setting expectations, motivating participation and introducing the problem. 
+
+The final method introduced in this chapter is Experience Diagraming. This is a method for exploring the current process and visualizing it. Each of our identified stakeholders will be utilized to help outline all the steps a warranty takes from creation to completion from their perspective. This lifecycle exploration helps us to see potential areas of improvement. It also provides a collection point to work on our context boundaries and ubiquitous language.
+
+## Chapter 3: Interviewing and Job Shadowing
+
+Now that we have identified all the stakeholders and processes we will impact, we can develop a picture of day to day operations. Job shadowing and interviewing are two valuable tools for exploring how the user does the task you are looking to augment every day. 
+
+In this chapter, we will outline how to conduct a successful interview and how to do job shadowing. This includes 
+
+- Using our stakeholder map and context bounding to identify interviewees.
```

Comment:
I like the itemized lists in some of the chapters describing the topics you'll dig into. Perhaps do this consistently across the outline?

Reply:
Reason: Makes it easier for editors/reviewers to scan through to see the scope.